lib/controller.js

```diff
@@ -946,11 +946,22 @@ controller.tesselFirmwareVerion = function(opts) {
         return updates.requestBuildList().then(function(builds) {
           // Figure out what commit SHA is running on it
           var version = updates.findBuild(builds, 'sha', versionSha).version;
-          logs.info('Tessel [' + tessel.name + '] version: ' + version);
+          var tesselVersion = 'Tessel [' + tessel.name + '] version: ' + version;
```

Comment:
I thought about adding the word firmware in there. I did add it. Then removed it before committing. Prefer it capitalized like you have it or lowercase?

Will get it updated with template strings when I get home.